ui/src/app/views/admin/info/add/info.add.component.ts

```diff
@@ -0,0 +1,52 @@
+import {Component, OnInit} from '@angular/core';
+import {ActivatedRoute, Router} from '@angular/router';
+import {AuthentificationStore} from '../../../../service/auth/authentification.store';
+import {Info} from '../../../../model/info.model';
+import {InfoService} from '../../../../service/info/info.service';
+import {InfoLevelService} from '../../../../shared/info/info.level.service';
+import {ToastService} from '../../../../shared/toast/ToastService';
+import {TranslateService} from '@ngx-translate/core';
+import {User} from '../../../../model/user.model';
+
+@Component({
+    selector: 'app-info-add',
+    templateUrl: './info.add.html',
+    styleUrls: ['./info.add.scss']
+})
+export class InfoAddComponent implements OnInit {
+    loading = false;
+    deleteLoading = false;
+    info: Info;
+    currentUser: User;
+    canAdd = false;
+    private infoLevelsList;
+
+    constructor(private _infoService: InfoService,
+                private _toast: ToastService, private _translate: TranslateService,
+                private _route: ActivatedRoute, private _router: Router,
+                private _authentificationStore: AuthentificationStore, _infoLevelService: InfoLevelService) {
+        this.currentUser = this._authentificationStore.getUser();
+        this.infoLevelsList = _infoLevelService.getInfoLevels()
+    }
+
+    ngOnInit() {
+        this._route.params.subscribe(params => {
+            this.info = new Info();
+        });
+    }
+
+    clickSaveButton(): void {
+        if (!this.info.title) {
+            return;
+        }
+
+        this.loading = true;
+        this._infoService.createInfo(this.info).subscribe( wm => {
```

Comment:
can you use .pipe(finalize(() => this.loading = false)) please and delete duplicate this.loading = false;